rocksdb_replicator/tests/rocksdb_replicator_test.cpp

```diff
@@ -348,6 +361,126 @@ TEST(RocksDBReplicatorTest, 1_master_2_slaves_chain) {
   EXPECT_EQ(db_slave_2->GetLatestSequenceNumber(), 2 * n_keys);
 }
 
+TEST(RocksDBReplicatorTest, 1_master_1_slave_upstream_itself) {
```

Comment:
Could you add a few lines of comments to explain:
- what is the purpose of test, and how would you plan to test it?
( I can get these info by reading the code, but, I think it might beneficial to briefly explain it)

Reply:
added